csrankings-y.csv

```diff
@@ -315,8 +315,7 @@ Yingwei Luo,Peking University,http://gis.pku.edu.cn/lab/people/lyw.htm,NOSCHOLAR
 Yingying Zhu,University of Texas at Arlington,https://zyy123jy.github.io/My-Web-Sites,PMjtni8AAAAJ
 Yingyu Liang,University of Wisconsin - Madison,http://pages.cs.wisc.edu/~yliang,_RVvnS4AAAAJ
 Yingzhen Li,Imperial College London,http://yingzhenli.net/home/en/?page_id=345,gcfs8N8AAAAJ
-Yinlong Xu,USTC,http://en.nhpcc.ustc.edu.cn/Peoples_9/201107/t20110701_115017.html,NOSCHOLARPAGE
-Yinqian Zhang,Ohio State University,https://cse.osu.edu/people/zhang.834-0,C7htwEIAAAAJ
+Yinlong Xu,USTC,http://en.nhpcc.ustc.edu.cn/Peoples_9/201107/t20110701_115017.html,NOSCHOLAR

```

Comment:
`NOSCHOLAR` needs to be `NOSCHOLARPAGE`.

If this is the same Yinqian Zhang (https://yinqian.org/), please update affiliation.